README.md

```diff
@@ -89,17 +141,22 @@ the header of the website when you scroll to the top of any page.
 
 ### Compatible with Git
 
-Jujutsu has two
-[backends](https://martinvonz.github.io/jj/latest/glossary#backend). One of them
-is a Git backend (the other is a native one [^native-backend]). This lets you
-use Jujutsu as an alternative interface to Git. The commits you create will look
-like regular Git commits. You can always switch back to Git. The Git support
-uses the [libgit2](https://libgit2.org/) C library.
+Jujutsu is designed so that the underlying data and storage model is abstract.
+Today, it features two [backends]—one of them uses the Git data model for
```

Comment:
nit: add spaces around the dash so it doesn't look like a hyphen

(and, as above, I think we should say "uses a Git repository for storage" instead of saying "uses the Git data model for storage")

Reply:
It's an em dash actually, which I think Ilya pointed out typically has no spaces. Will take care of the Git wording.